client/src/components/Map/index.tsx

```diff
@@ -43,7 +49,7 @@ function renderLandingMap(nodes) {
             to={`/learn/${node.superBlock}/`}
           >
             <div style={linkSpacingStyle}>
-              {generateIconComponent(node.superBlock, 'map-icon')}
+              {generateIconComponent(node.superBlock as SuperBlock, 'map-icon')}
```

Comment:
required because generateIconComponent define superBlock as a separate type not string